src/Sylius/Bundle/CoreBundle/Fixture/Factory/AddressExampleFactory.php

```diff
@@ -135,15 +137,11 @@ public function create(array $options = [])
         $this->assertCountryCodeIsValid($options['country_code']);
         $address->setCountryCode($options['country_code']);
 
-        if (null !== $options['province_code']) {
-            $this->assertProvinceCodeIsValid($options['province_code'], $options['country_code']);
-            $address->setProvinceCode($options['province_code']);
-        } else {
-            $this->assertCountryHasNoProvinces($options['country_code']);
-            $address->setProvinceName($options['province_name']);
-        }
+        $this->resolveCountryProvince($options, $address);
 
-        $options['customer']->addAddress($address);
+        if (isset($options['customer'])) {
+            $options['customer']->addAddress($address);
```

Comment:
We allow for `null` value, shouldn't we verify we don't call this method on `null` object?